test/dynamo/test_aot_autograd.py

```diff
@@ -867,7 +867,7 @@ def _prepare_model_args():
                 """\
 SeqNr|OrigAten|SrcFn
 0|aten.convolution.default|l__self___conv1
-0|aten.add.Tensor|l__self___bn1
+0|aten.add.Tensor|l__self___bn1_1
```

Comment:
I assume my pr is likely get reverted couple times. so it should be better for me to do something similar to
```
xx if config._dynamo.use_single_step_graph else xxx.
```

Reply:
Another approach is to add a normalization function to the string which will remove small differences like this on SrcFn

Reply:
sg, I have it working locally, don't want to spam the ci machine so I will submit it with my next batch of changes.